<?xml version="1.0"?>
<launch>
    <arg name="startState" default="takeoff"/>

    <param name="uavName" type="string" value="suav"/>
    <param name="ON_GROUND" type="bool" value="false"/>
    <param name="startState" type="string" value="$(arg startState)"/>
    <param name="ignoreSearch" type="bool" value="true"/>
    <param name="hold_time_for_locateUSV" type="double" value="20.0"/>

    <node pkg="suav_control" type="suav" name="suav" output="screen"></node>
</launch>
